src/rgb/carbonado.rs

```diff
@@ -13,14 +16,32 @@ use crate::{
 #[derive(Debug, Clone, Eq, PartialEq, Display, From, Error)]
 #[display(doc_comments)]
 pub enum StorageError {
+    /// File '{0}' retrieve causes error. {1}
+    FileRetrive(String, String),
+    /// File '{0}' write causes error. {1}
+    WriteRetrive(String, String),
+    /// Changes '{0}' retrieve causes error. {1}
+    ChangesRetrive(String, String),
+    /// Changes '{0}' write causes error. {1}
+    ChangesWrite(String, String),
+    /// Fork '{0}' write causes error. {1}
+    ForkWrite(String, String),
+    /// Merge '{0}' write causes error. {1}
+    MergeWrite(String, String),
     /// Retrieve '{0}' strict-encoding causes error. {1}
     StrictRetrive(String, String),
     /// Write '{0}' strict-encoding causes error. {1}
     StrictWrite(String, String),
+    /// Retrieve '{0}' serialize causes error. {1}
+    SerializeRetrive(String, String),
+    /// Write '{0}' serialize causes error. {1}
+    SerializeWrite(String, String),
     /// Retrieve '{0}' carbonado causes error. {1}
     CarbonadoRetrive(String, String),
```

Comment:
The word Retrieve is misspelled in FileRetrieve, WriteRetrieve, ChangesRetrieve, StrictRetrieve, SerializeRetrieve, and CarbonadoRetrieve.